kolibri/plugins/coach/assets/src/views/plan/QuizSummaryPage/index.vue

```diff
@@ -5,66 +5,51 @@
     :authorized="$store.getters.userIsAuthorizedForCoach"
     authorizedRole="adminOrCoach"
     :showSubNav="true"
+    maxMainWidth="1440px"
   >
 
     <TopNavbar slot="sub-nav" />
-
-    <KPageContainer v-if="!loading">
-      <section>
-        <BackLinkWithOptions>
-          <BackLink
-            slot="backlink"
-            :to="$router.getRoute('EXAMS')"
-            :text="coachString('allQuizzesLabel')"
-          />
+    <KGrid gutter="16">
```

Comment:
Recommend setting consistent vertical and horizontal gutters:

![image](https://user-images.githubusercontent.com/2367265/67825583-0735b500-fa87-11e9-9ff1-16fa06546ad9.png)

Can do this by adding a new `topMargin` prop to `KPageContainer`:

https://github.com/learningequality/kolibri/blob/develop/packages/kolibri-components/src/KPageContainer.vue#L49

Reply:
Good eye @indirectlylit - I've added the prop and used the same default that was in the CSS on KPageContainer - so now I'm overriding it to use the same value as my gutters.